manifold/src/boolean_result.cpp

```diff
@@ -636,20 +637,20 @@ Manifold::Impl Boolean3::Result(Manifold::OpType op) const {
   // next one knows where to slot in.
   VecDH<int> facePtrR = faceEdge;
   // Intersected halfedges are marked false.
-  VecDH<bool> wholeHalfedgeP(inP_.halfedge_.size(), true);
-  VecDH<bool> wholeHalfedgeQ(inQ_.halfedge_.size(), true);
+  VecDH<char> wholeHalfedgeP(inP_.halfedge_.size(), true);
+  VecDH<char> wholeHalfedgeQ(inQ_.halfedge_.size(), true);
```

Comment:
I suppose because CUDA doesn't support addressing single bits? Works for me.

Reply:
No, this is due to std::vector<T> is being used for the cache, and you know the problem with std::vector<bool>